### PR TITLE
[9.0][FIX] add button to product form

### PR DESCRIPTION
**Pull request by SodexisTeam**
Added the potential quantity button on product form too.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can't this be put in the original module (stock_available) in the view that is available both for template and product?

**Comment by atchuthan** [comment]
no, the field "potential_qty" is added only in stock_available_mrp module.

**Comment by pedrobaeza** [comment]
Anyway, my point is that there's a view that is valid for both template and product. I'm asking about adding there the field for not having to duplicate the work each time, in whatever module you need to do it.

**Comment by atchuthan** [comment]
@pedrobaeza Moved the field to stock_available module as per our previous discussion.

change label to "needs review"

File: stock_available/views/product_template_view.xml [hunk 1]
```diff
@@ -20,6 +19,15 @@
                             <span class="o_stat_text">Available</span>
                         </div>
                     </button>
+                    <button type="action" name="%(stock.product_open_quants)d"
+                        attrs="{'invisible':[('type', '!=', 'product')]}"
```
Comment (pedrobaeza) [inline stock_available/views/product_template_view.xml]: What about consumable products?

Comment (atchuthan) [reply stock_available/views/product_template_view.xml]: I tried to use the same visibility as "immediately_usable_qty" field
https://github.com/OCA/stock-logistics-warehouse/pull/192/files/70c8ae8602260ef375d4c765a5d7a02e02eb3e34#diff-33210f15e1b07dbc688105994395937aR14

Comment (pedrobaeza) [reply stock_available/views/product_template_view.xml]: Yeah, but both can make sense also for consumable products I think

Comment (atchuthan) [reply stock_available/views/product_template_view.xml]: Yes, I agree.

**Review by pedrobaeza — APPROVED** [review]
Squashing on merge

File: stock_available/models/product_product.py [hunk 2]
```diff
@@ -29,11 +29,27 @@ def _immediately_usable_qty(self):
...
-        compute='_immediately_usable_qty',
+        compute='_compute_immediately_usable_qty',
         string='Available to promise',
         help="Stock for this Product that can be safely proposed "
              "for sale to Customers.\n"
              "The definition of this value can be configured to suit "
              "your needs")
+    potential_qty = fields.Float(
+        compute='_compute_potential_qty',
+        type='float',
```
Comment (Garamotte) [inline stock_available/models/product_product.py]: You don't need to define the type attribute on new API.

File: stock_available/models/product_template.py [hunk 3]
```diff
@@ -24,11 +24,36 @@ def _immediately_usable_qty(self):
...
-        compute='_immediately_usable_qty',
+        compute='_compute_immediately_usable_qty',
         string='Available to promise',
         help="Stock for this Product that can be safely proposed "
              "for sale to Customers.\n"
              "The definition of this value can be configured to suit "
              "your needs")
+    potential_qty = fields.Float(
+        compute='_compute_potential_qty',
+        type='float',
```
Comment (Garamotte) [inline stock_available/models/product_template.py]: You don't need to define the type attribute on new API.

File: stock_available_mrp/__openerp__.py [hunk 4]
```diff
@@ -5,15 +5,14 @@
...
               u"Odoo Community Association (OCA)",
     'category': 'Hidden',
     'depends': [
         'stock_available',
         'mrp'
     ],
-    'data': [
-        'views/product_template_view.xml',
-    ],
+    'data': [],
```
Comment (Garamotte) [inline stock_available_mrp/__openerp__.py]: Please remove empty key.

**Review by Garamotte — COMMENTED** [review]
Some comments, but not blocking.

**Comment by atchuthan** [comment]
@pedrobaeza squashed all commits
@sylvain-garancher changes done

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**